btcec/signature.go

```diff
@@ -307,10 +340,10 @@ func recoverKeyFromSignature(curve *KoblitzCurve, sig *Signature, msg []byte,
 	// We calculate the two terms sR and eG separately multiplied by the
 	// inverse of r (from the signature). We then add them to calculate
 	// Q = r^-1(sR-eG)
-	invr := new(big.Int).ModInverse(sig.R, curve.Params().N)
+	invr := new(big.Int).ModInverse(sigR, curve.Params().N)
```

Comment:
Any reason inverse isn't being done in the field where it's a lot faster and then converted to a `big.Int`?